lib/xlsx/xform/sheet/cell-xform.js

```diff
@@ -251,17 +266,35 @@ utils.inherits(CellXform, BaseXform, {
         this.currentNode = 'v';
         return true;
 
+      case 't':
+        this.currentNode = 't';
+        return true;
+
+      case 'r':
+        this.parser = richTextXForm;//this.map.r;
```

Comment:
parsing is state-full so this will cause problems if more than one parse is running.

Reply:
Good point! See fix below...